addons/website/views/snippets/s_carousel.xml

```diff
@@ -32,7 +32,7 @@
                     <div class="container oe_unremovable">
                         <div class="row">
                             <div class="carousel-content col-lg-8 offset-lg-2 bg-black-50 text-center pt48 pb40">
-                                <h2 style="font-size: 62px;">Clever Slogan</h2>
+                                <h2 class="display-3">Clever Slogan</h2>
```

Comment:
`<h2 class="display-3">` is actually not a thing I though deeply of :grimacing: This is indeed what you need here: semantically an h2 but with the style of a display-3... but a bit weird because it will use the color of an h2 while `<h2 class="h3">` would use the color of an h3 not an h2 :grimacing: 

And in this case, the UI shows "header 2" in both case... which is good because we want to show the semantic meaning for the font-style... but the color would still be inconsistent :grimacing: Maybe it is not a big deal...

I am kinda wondering if we should not use `display-3-fs` instead though. Or maybe handle the case of `<hx class="hy/display-z">` to show something different than "Header X" as font-style in the UI.
What was your thought on this?

Reply:
Let's use `*-fs` classes :+1: